### DISCUSSION TITLE
Help finding out which packages have been installed on drive that no longer boots

### DISCUSSION THREAD

**Discussion by designet** [Homebrew/discussions#2708] [Getting started] [answered]
My old standby 2012 MBP failed when I installed a new driver - and it will no longer boot. I took the opportunity to start over with a brand new machine. I decided to start from scratch (not use Apple Migration Assistant). Just did a fresh install of homebrew on the new machine. Now I'd like to selectively install the same packages that were on the old machine. Is there a place I can look on the old drive to find the list of installed packages? I've made a backup of the old drive on a USB drive and I can read and copy whatever files I need from that. Thanks!

**Comment by SMillerDev** [comment, marked as answer]
Assuming the machine was using an Intel CPU: `ls /usr/local/Cellar` shows all installed packages.

**Reply by designet** [reply]
Thanks @SMillerDev ! There it is. Now I'm looking at the folder structure on the new machine, and I see that no /usr directory has even been created on this machine yet. A bit of a surprise.

**Reply by SMillerDev** [reply]
If the new machine is an ARM based machine it'll be in `/opt/homebrew`

**Reply by designet** [reply]
It's an i9.

**Reply by designet** [reply]
I'm not seeing system directories in the Finder UI. All there in terminal. Must be some permission I need to set.

**Reply by designet** [reply]
Thanks for your help.

**Reply by alebcay** [reply]
I believe the `/usr` folder is hidden by default in the Finder UI. In Finder, you may be able to access via the menu bar - Go > Go to Folder (<kbd>⌘</kbd>+<kbd>⇧</kbd>+<kbd>G</kbd>). Then you can navigate directly to `/Volumes/<USB drive>/usr/...` to find what was left in the backup.

**Reply by designet** [reply]
@alebcay that sounds right. I'll give it a try.
